example/follow_me/CMakeLists.txt

```diff
@@ -2,16 +2,26 @@ cmake_minimum_required(VERSION 2.8.12)
 
 project(follow_me)
 
-if (MSVC)
-    find_package(Boost 1.64 COMPONENTS system date_time REQUIRED )
+if(NOT MSVC)
+    add_definitions("-std=c++11 -Wall -Wextra -Werror")
+	# Line below required if /usr/local/include is not in your default includes
+    #include_directories(/usr/local/include)
+    # Line below required if /usr/local/lib is not in your default linker path
+    #link_directories(/usr/local/lib)
+    find_package(Boost 1.58 COMPONENTS system REQUIRED )
+else()
     include_directories(${CMAKE_SOURCE_DIR}/../../install/include)
     link_directories(${CMAKE_SOURCE_DIR}/../../install/lib)
     add_definitions("-std=c++11 -WX -W2")
+    find_package(Boost 1.64 COMPONENTS system date_time REQUIRED )
```

Comment:
`date_time` required, really?

Reply:
You'll have to ask @shakthi-prashanth-m - I moved things around for consistency, but this was already here.

Reply:
Oh, you're right.